src/Yaapii.Xml/XMLSlice.cs

```diff
@@ -35,23 +35,23 @@
 namespace Yaapii.Xml
 {
     /// <summary>
-    /// A XML which will slice when you call Nodes(xpath).
+    /// An xml object which will execute the slice operation once its method is invoked.

```

Comment:
I think this change shifts the comment away from the EO paradigm which suggests to see an object as a whole and not focus on its technical inner workings.

Reply:
I see. I have undone the change